package.xml

```diff
@@ -6,8 +6,17 @@
   <description>gtsam</description>
 
   <maintainer email="gtsam@lists.gatech.edu">Frank Dellaert</maintainer>
+
+  <!-- Main License for GTSAM sources -->
   <license>BSD</license>
```

Comment:
```suggestion
  <license>BSD-3-Clause</license>
```

Although `BSD` usually indicates the 3 Clause BSD license within the ROS community I think explicitly using the SPDX.org identifier for it is an improvement I recommend for every ROS package.

Reply:
exactly, I saw somewhere in the ROS docs that BSD = 3-clause BSD. Anyway, as long as your more explicit names do not raise any warning in any ROS tool or linter, I also like them more. 
Thanks!